product-group-app-ktor/build.gradle.kts

```
@@ -80,6 +75,12 @@ kotlin {
                 implementation(project(":product-group-common"))
                 implementation(project(":product-group-mapper"))
                 implementation(project(":product-group-stubs"))
+                implementation(project(":product-group-log-mapper"))
+                implementation(project(":product-group-api-log"))
+                implementation(project(":product-group-common-logging"))
```

```suggestion
                implementation(project(":product-group-logging-common"))
```
Когда модулей много накапливается, очень тяжело в них ориентироваться, если они в разнобой. 
Поэтому одна группа модулей должна именоваться так, чтоб все модули в ней были рядышком